src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -139,6 +143,9 @@ public void renewClusterSlots(Connection jedis) {
 
   private void discoverClusterSlots(Connection jedis) {
     List<Object> slotsInfo = executeClusterSlots(jedis);
+    if (slotsInfo.isEmpty()) {
+      return;
```

Comment:
```suggestion
      throw new JedisClusterException("Cluster slots list is empty");
```

Reply:
throw JedisClusterException, catch and ignore it, and try next nodes.

Reply:
Fixed